### PR TITLE
[DEVC-1215][ESD-187] Skylark with HTTP2

### PR DESCRIPTION
**Pull request by pmiettinen**
Requires https://github.com/swift-nav/piksi_firmware_private/pull/2568

### CODE REVIEW

File: package/libpiksi/libpiksi/src/version.c [hunk 1]
```diff
@@ -1,106 +1,106 @@
...
-  }
-
-  return a->patch - b->patch;
-}
-
-int version_devstr_cmp(const piksi_version_t *a, const piksi_version_t *b)
-{
-  return strcmp(a->devstr, b->devstr);
-}
+/*
```
Comment (pmiettinen) [inline package/libpiksi/libpiksi/src/version.c]: This file had CRLF line changes, changed to LF only.

**AND** updated my editor default linefeed to LF only.

File: package/libnetwork/libnetwork/src/libnetwork_http2.c [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+#include <curl/curl.h>
+
+#include <libpiksi/logging.h>
+
+#include <libnetwork/libnetwork.h>
+#include <libnetwork/libnetwork_http2.h>
+
+#define TIMEOUT_MAX_S 1
+
+static void dump(const char *text, FILE *stream, unsigned char *ptr, size_t size)
```
Comment (pmiettinen) [inline package/libnetwork/libnetwork/src/libnetwork_http2.c]: These debug functions can be removed from the final version.

File: package/libnetwork/libnetwork/include/libnetwork/libnetwork.h [hunk 3]
```diff
@@ -72,6 +74,18 @@ typedef enum {
 
 const char *libnetwork_status_text(network_status_t status);
 
+void network_ctx_setup(network_context_t *ctx);
```
Comment (pmiettinen) [inline package/libnetwork/libnetwork/include/libnetwork/libnetwork.h]: Needs function docs.

**Comment by benjaminaltieri** [comment]
closing since this was not required after all

**Pull request closed without merging**